horace_core/utilities/@sqw/combine_sqw.m

```diff
@@ -105,7 +109,8 @@
 
 for i=1:ndims1
     step=wout.data.p{i}(2)-wout.data.p{i}(1);
-    wout.data.p{i}=[min_full{i}-step+eps:step:max_full{i}+step-eps]';
+    wout.data.p{i}=make_const_bin_boundaries([min_full{i},step,max_full{i}]);
+    %[[min_full{i}-step+eps:step:max_full{i}+step-eps]';
```

Comment:
dead code

Reply:
The code  there was different from the one it replaces. Testing there is poor so I do not know what Is correct. I've added description about this hesitation and hope to clarify this issue in symmetrization ticket.